src/components/graph-view.js

```diff
@@ -188,16 +196,19 @@ class GraphView extends React.Component<IGraphViewProps, IGraphViewState> {
       .call(this.zoom);
 
     this.selectedView = d3.select(this.view);
-
+    if(this.props.initialBBox)
+      this.handleZoomToFitImpl(this.props.initialBBox, 0);
     // On the initial load, the 'view' <g> doesn't exist until componentDidMount.
     // Manually render the first view.
     this.renderView();
 
-    setTimeout(() => {
-      if (this.viewWrapper != null) {
-        this.handleZoomToFit();
-      }
-    }, this.props.zoomDelay);
+    if (!this.props.initialBBox) {
+      setTimeout(() => {
+        if (this.viewWrapper != null) {
+          this.handleZoomToFit();
+        }
+      }, this.props.zoomDelay);
+    }
```

Comment:
You should choose to perform one of these actions as a default, and one inside an if statement.

```
if (initialBBox) {
  this.handleZoomToFitImpl(initialBBox, 0);
  return;
}

setTimeout(() => {
  if (this.viewWrapper != null) {
    this.handleZoomToFit();
  }
}, this.props.zoomDelay); // needs destructuring too
```